foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/QueryResponse.java

```diff
@@ -28,12 +28,18 @@
  */
 public class QueryResponse implements ActionResponse {
     private List<Document> documents;
+    private long totalHits;
 
     public QueryResponse() {
     }
 
     public QueryResponse(List<Document> documents) {
```

Comment:
@r0goyal, move it to action response? then how would you represent this for
say for example nested groupings?

It should be used only as an hint for pagenation, which we don't use in
anything other than queries.

----
Upset the established order, and everything becomes chaos. I'm an agent of
chaos. Oh and you know the thing about chaos, it's fair.

On Wed, Dec 9, 2015 at 12:41 PM, Rishabh <notifications@github.com> wrote:

> In
> foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/QueryResponse.java
> <https://github.com/Flipkart/foxtrot/pull/99#discussion_r47057219>:
>
> > @@ -28,12 +28,18 @@
> >   */
> >  public class QueryResponse implements ActionResponse {
> >      private List<Document> documents;
> > +    private long totalHits;
> >
> >      public QueryResponse() {
> >      }
> >
> >      public QueryResponse(List<Document> documents) {
>
> This method can be removed since we should always have total search hits.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Flipkart/foxtrot/pull/99/files#r47057219>.
>